### DISCUSSION TITLE
Will ImageMagick compile with php 8.4?

### DISCUSSION THREAD

**Discussion by BowFarmer** [ImageMagick/ImageMagick#7805] [General]
I've been testing php 8.4 on a VM I use for a development version of Wordpress, and I can't get ImageMagick to compile with php 8.4. Is this a known issue?

**Comment by dlemstra** [comment]
I guess you are talking about IMagick instead of ImageMagick. You should probably ask that question here instead: https://github.com/Imagick/imagick?

**Reply by BowFarmer** [reply]
Thank you.
